nix/modules/happs.nix

```diff
@@ -159,6 +156,7 @@ in
         buildInputs = zomeDeps ++ [
           # Need `hc` to package DNAs and hApps.
           inputs'.holonix.packages.holochain
```

Comment:
This line should be removed, `hc` used to be included in the `holochain` package but is now separate. We don't need the `holochain` binary here at all.

Reply:
fixed abce12bf3c229481780dec764b3f8ab04011ff53